test/integration/scheduler/extender_test.go

```diff
@@ -313,43 +313,38 @@ func TestSchedulerExtender(t *testing.T) {
 		extender3.serveHTTP(t, w, req)
 	}))
 	defer es3.Close()
-
-	policy := schedulerapi.Policy{
-		ExtenderConfigs: []schedulerapi.ExtenderConfig{
-			{
-				URLPrefix:      es1.URL,
-				FilterVerb:     filter,
-				PrioritizeVerb: prioritize,
-				Weight:         3,
-				EnableHTTPS:    false,
-			},
-			{
-				URLPrefix:      es2.URL,
-				FilterVerb:     filter,
-				PrioritizeVerb: prioritize,
-				BindVerb:       bind,
-				Weight:         4,
-				EnableHTTPS:    false,
-				ManagedResources: []schedulerapi.ExtenderManagedResource{
-					{
-						Name:               extendedResourceName,
-						IgnoredByScheduler: true,
-					},
+	policyString := fmt.Sprintf(`{
```

Comment:
It could be better to use a struct and marshal it into JSON value.

Reply:
Do you mean having a test struct for example (testPolicy) that mimics `schedulerapi.Policy` then pass the JSON string to `initTestScheduler`?

Reply:
I could also use a `map[string]interface{}` and marshal it into JSON.